node.go

```diff
@@ -14,9 +14,9 @@ type Node struct {
 	Host    Host
 	aliases []Host
 
-	cluster       *Cluster
-	pool          *Pool
-	refreshTicker *time.Ticker
+	cluster         *Cluster
+	pool            *Pool
+	refreshDoneChan chan bool
```

Comment:
You could use a `chan struct{}` here instead. Not much difference but theres no reason not to.